packages/terra-core-docs/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Updated
+  * Updated terra signature example to include text and image signature.
+
 * Changed
   * Fixed typos and imports in `terra-list` docs.
```

Comment:
nit: for consistency, can we change this heading to "Changed" and combine it with the changelog entry on line 8? also can we add backticks around terra-signature?

```suggestion
* Changed
  * Fixed typos and imports in `terra-list` docs.
  * Updated `terra-signature` example to include text and image signature.
```